src/fastga_he/powertrain_builder/powertrain.py

```diff
@@ -1414,6 +1387,20 @@ def get_fuel_tank_list(self) -> Tuple[list, list]:
 
         return components_names, components_types
 
+    def get_generator_list(self) -> list:
```

Comment:
Why do we need a list of generators ?

Reply:
Ok I understand now. But I think it may be simplified. Do you hink it is safe to say that if the ICE (or turbohaft or high rpm ICE) is not connected to a generator then it is a propulsive load ?